pages/billing/subscription.en-US.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Subscription
+ogImageTitle: Subscription
+ogImageSubtitle: Subscription on Zeabur
+---
+
+# Subscription
+
+## What is Subscription?
+
+There're 3 types of plans on Zeabur: **Free Plan**, **Developer Plan** and **Team Plan**.
```

Comment:
Usually we prefer to express “there are” as “there are” instead of the simplified version.